### DISCUSSION TITLE
Neigungswinkel von Jalousien

### DISCUSSION THREAD

**Discussion by Mate883** [brutella/hkknx-public#332] [General]
Hi zusammen,

in HKKNX habe ich bisher keine Möglichkeit gefunden den Neigungswinkel meiner Jalousien einzustellen. Es ist möglich die Gruppenadresse der Neigung beim Gerät Jalousie zu hinterlegen (HomeKit erkennt das, wenn auch mehr schlecht als recht) aber in HKKNX kann ich die Neigung nicht einstellen....wird da evtl. noch was implementiert?

**Comment by brutella** [comment]
Korrekt, in hkknx gibt es dazu noch keine UI dazu – wird im einem zukünftigen Update behoben.  → #333
